src/test/java/pages/BasePage.java

```diff
@@ -0,0 +1,44 @@
+package pages;
+
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+
+public class BasePage ( WebDriver givenDriver) {
+
+    driver = givenDriver;
+    wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+    PageFactory.initElements(driver,this);
+}
+
+    protected static WebDriver driver;
+    private WebDriverWait wait;
+    protected Actions actions;
+
+    public BasePage(WebDriver givenDriver) {
+        driver = givenDriver;
+        wait = new WebDriverWait(driver, Duration.ofSeconds(3));
+        actions = new Actions(driver);
+    }
+
+    //public static WebDriver driver = null;
+
+
+    //public static String url = null;
+
+    //String playlistName = ":)";
+
+    // public BasePage(WebDriver givenDriver) {
+    // driver = givenDriver;
+    // WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+    // Actions actions = new Actions(driver);
+    // }
+
+
+
+}
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/225709032-1a8df83f-3883-434d-80e4-9bda1020e2be.png)